include/ppx/application.h

```diff
@@ -242,6 +242,14 @@ struct ApplicationSettings
             grfx::Format depthFormat = grfx::FORMAT_UNDEFINED;
             uint32_t     imageCount  = 2;
         } swapchain;
+
+#if defined(PPX_BUILD_XR)
+        struct
+        {
+            XrVector3f  pos;
+            XrExtent2Df size;
+        } ui;
```

Comment:
Can you add a comment on what these coordinates represent? E.g. is `pos` the upper-left corner, center point, or something else?

Reply:
fixed